cvat-ui/src/actions/boundaries-actions.ts

```diff
@@ -55,9 +64,23 @@ export function resetAfterErrorAsync(): ThunkAction {
 
                 await job.logger.log(LogType.restoreJob);
 
-                dispatch(boundariesActions.resetAfterError(job, states, frameNumber, frameData, minZ, maxZ, colors));
+                dispatch(boundariesActions.resetAfterError(
+                    job,
+                    states,
+                    state.annotation.job.openTime || Date.now(),
+                    frameNumber,
+                    frameData.filename,
+                    frameData.hasRelatedContext,
+                    colors,
+                    [],
+                    frameData,
+                    minZ,
+                    maxZ,
+                ));
             } else {
-                dispatch(boundariesActions.resetAfterError(null, [], 0, null, 0, 0, []));
+                dispatch(boundariesActions.resetAfterError(
+                    null, [], null, 0, '', false, [], [], null, 0, 0,
```

Comment:
Why not to use default params for `resetAfterError` action instead of specifying lots of inconvinient default values like null, 0, false etc.?

Reply:
I looked at the code, no need to have ``else`` branch if job is ``undefined`` because this code can not be called in this case.